lib/commanded/process_managers/process_manager_instance.ex

```diff
@@ -184,45 +191,56 @@ defmodule Commanded.ProcessManagers.ProcessManagerInstance do
 
     %State{idle_timeout: idle_timeout} = state
 
+    telemetry_metadata = telemetry_metadata(event, state)
+    start_time = telemetry_start(telemetry_metadata)
+
     case handle_event(event, state) do
-      {:error, _error} = error ->
+      {:error, error} ->
         failure_context = build_failure_context(event, context, nil, state)
 
-        handle_event_error(error, event, failure_context, state)
+        telemetry_stop(start_time, telemetry_metadata, {:error, error})
+
+        handle_event_error({:error, error}, event, failure_context, state)
 
       {:error, error, stacktrace} ->
         failure_context = build_failure_context(event, context, stacktrace, state)
 
-        handle_event_error({:error, error}, event, failure_context, state)
+        telemetry_stop(start_time, telemetry_metadata, {:error, error, stacktrace})
 
-      {:stop, _error, _state} = reply ->
```

Comment:
I'm not sure why this was here previously, it's not part of the `handle/2` API

Reply:
This would match the `{:stop, reason :: term(), new_state}` return value for [`handle_cast/2`](https://hexdocs.pm/elixir/GenServer.html#c:handle_cast/2) where the error is given as the reason to stop the process.